qiita_db/processing_job.py

```diff
@@ -1007,75 +1007,74 @@ def release(self):
 
     def release_validators(self):
         """Allows all the validator job spawned by this job to complete"""
-        with qdb.sql_connection.TRN:
```

Comment:
The main change here is moving the `with qdb.sql_connection.TRN:` to line 1074 as there is the only place actually needed. 

This change was needed as the updates to the self.step (line 1030) was not being reflected in the GUI until everything was done due to transaction encapsulation - AKA useless.